cmd/kubeadm/app/preflight/checks.go

```diff
@@ -1063,7 +1063,7 @@ func RunRootCheckOnly(ignorePreflightErrors sets.String) error {
 func RunPullImagesCheck(execer utilsexec.Interface, cfg *kubeadmapi.InitConfiguration, ignorePreflightErrors sets.String) error {
 	containerRuntime, err := utilruntime.NewContainerRuntime(utilsexec.New(), cfg.NodeRegistration.CRISocket)
 	if err != nil {
-		return err
+		return &Error{Msg: err.Error()}
```

Comment:
this change seems fine.

a number of `/cmd` instances call `RunPullImagesCheck` directly, which means that they should return `preflight.Error`
https://github.com/kubernetes/kubernetes/search?q=RunPullImagesCheck

i see that other exported functions like `RunInitNodeChecks` and `RunJoinNodeChecks` call the `RunChecks` function which returns `Error{}` (correctly), are there any similar cases to `RunPullImagesCheck` or is it the only one?

Reply:
This seems to be the only one in this package